src/lib/mongoBackend/mongoUpdateContext.cpp

```diff
@@ -74,6 +74,7 @@ HttpStatusCode mongoUpdateContext
       std::string details = std::string("service path length ") + lenV + " is greater than the one in update";
       alarmMgr.badInput(clientIp, details);
       responseP->errorCode.fill(SccBadRequest, "service path length greater than one in update");
+      responseP->oe.fill(SccBadRequest, "service path length greater than one in update", "BadRequest");
```

Comment:
Fixed in 3e4cd945bb1ab64ca9dc5507568445f5241b8c32 (surprisingly it seems that no adaptation was needed in any .test).